javascript/javascript1/week1/Goodboy-Oldboy.js

```diff
@@ -0,0 +1,18 @@
+
+//Goodboy-Oldboy (A dog age calculator)
+
+let dogYearOfBirth = 2010;
+let dogYearFuture = 2027;
+let dogYear=dogYearFuture-dogYearOfBirth;
+let dogOld=dogYear*7 
```

Comment:
There's some inconsistencies in your formatting - eg. some lines end in `;` while others don't. Make sure you install the "Prettier" extension and set it up to auto format on save:
```
- Open VSCode Settings
- Search 'format'
- Change 'Editor: Default Formatter' to 'Prettier - code formatter'
- Activate 'Editor: Format on save'
```
This will make sure everything looks neat and tidy and consistent :)